src/ansys/fluent/core/solver/flobject.py

```diff
@@ -169,6 +171,23 @@ def to_python_name(fluent_name: str) -> str:
     return name
 
 
+def _get_root_from_child_object(obj):
+    root = obj._parent
+    if root is None:
+        root = obj
+    else:
+        root = _get_root_from_child_object(root)
+    return root
+
+
+def _interrupt(obj):
+    if pyfluent.SUPPORT_SOLVER_INTERRUPT:
+        root = _get_root_from_child_object(obj)
```

Comment:
Don't we also need to identify that the command is an "interruptable" command? - which we can determine from a hard-coded list (for now).

Reply:
Done.